src/NLog/LayoutRenderers/ProcessInfoLayoutRenderer.cs

```diff
@@ -69,6 +69,12 @@ public ProcessInfoLayoutRenderer()
         public ProcessInfoProperty Property { get; set; }
```

Comment:
@304NotModified Think it should just have DefaultValue, and not DefaultParameter. Make PR to remove DefaultParameter?